server/routerlicious/packages/routerlicious-base/src/utils/sessionHelper.ts

```diff
@@ -281,13 +281,27 @@ export async function getSession(
 	sessionStickinessDurationMs: number = defaultSessionStickinessDurationMs,
 	messageBrokerId?: string,
 	clusterDrainingChecker?: IClusterDrainingChecker,
+	ephemeralDocumentTTLSec?: number,
 ): Promise<ISession> {
 	const lumberjackProperties = getLumberBaseProperties(documentId, tenantId);
 
 	const document: IDocument = await documentRepository.readOne({ tenantId, documentId });
 	if (!document || document.scheduledDeletionTime !== undefined) {
 		throw new NetworkError(404, "Document is deleted and cannot be accessed.");
 	}
+	if (document.isEphemeralContainer && ephemeralDocumentTTLSec !== undefined) {
+		// Check if the document is ephemeral and has expired.
+		const currentTime = Date.now();
+		const ephemeralDocumentExpiryTime = document.createTime + ephemeralDocumentTTLSec * 1000;
+		if (currentTime > ephemeralDocumentExpiryTime) {
+			throw new NetworkError(
+				404,
+				`Ephemeral Container Expired: ${Math.floor(
```

Comment:
Same as above. Could the time since expired be moved to properties?

Reply:
This is for the Network error returned to the client. There isn't a way to add generic properties. Do you feel that this is too verbose for the client, and `Ephemeral Container Expired` is sufficient?